docs/en/latest/practices/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md

```diff
@@ -0,0 +1,113 @@
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Observability (monitoring functionality) has always played an important role in system maintenance. A good monitoring system can help engineers quickly understand the status of services running in production environments and can locate problems or give early warning of anomalies when they occur.
+
+Prometheus is a leading open-source project focused on metrics and alerting that has changed the way the world does monitoring and observability. And Apache APISIX has enhanced its support for Prometheus Metrics in recent releases, adding a new feature for use in conjunction with the `public-api` plugin. This article will introduce how to configure `public-api` to protect Prometheus to collect Apache APISIX 's Metrics data.
+
+## Initial Knowledge about `public-api`
+
+When users develop custom plugins in Apache APISIX, they can define some APIs (hereinafter referred to as public API) for the plugins. In practical application scenarios, the provided interface is for internal calls, rather than being open on the public network for anyone to call.
+
+Therefore, Apache APISIX has designed a `public-api` plugin. With this plugin, you can solve the pain points of using the public API. You can set a custom URI for the public API and configure any type of plugin. For more information about `public-api`, see the [public-api](https://apisix.apache.org/docs/apisix/plugins/public-api/) plugin's document.
+
+The main role of the `public-api` plugin in this document is to protect the URI exposed by Prometheus.
+
+**Note**: One thing we should note is that this feature is only available in APISIX version 2.13 and later.
+
+## Begin to access Apache APISIX Prometheus Metrics
+
+### Step1: Install APISIX Ingress Controller
+
+First, we deploy Apache APISIX, ETCD, and APISIX Ingress Controller to a local Kubernetes cluster via Helm.
+
+```sh
+helm repo add apisix https://charts.apiseven.com
+helm repo update
+kubectl create namespace ingress-apisix
+helm install apisix apisix/apisix --namespace ingress-apisix \
+ --set ingress-controller.enabled=true
+```
+
+After installation, please wait until all services are up and running. Specific status confirmation can be checked with the following command.
+
+```sh
+kubectl get all -n ingress-apisix
```

Comment:
Do we have any dedicated docs to introduce the way we install Apache APSIX Ingress Controller? @tao12345666333 

I don't think show the installation in a lot of docs is a good idea.

Reply:
Yeah, we have!
"The hard way": https://apisix.apache.org/docs/ingress-controller/practices/the-hard-way

Reply:
I have bought the Grammarly package and have taken a lot of its advice.
Thanks for your great suggestions.